src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -215,14 +215,55 @@ type TaskSeq =
 
     /// <summary>
     /// Combines the given task sequence of task sequences and concatenates them end-to-end, to form a
-    /// new flattened, single task sequence. Each task sequence is awaited item by item, before the next is iterated.
+    /// new flattened, single task sequence, like <paramref name="TaskSeq.collect id"/>. Each task sequence is
+    /// awaited and consumed in full, before the next one is iterated.
     /// </summary>
     ///
     /// <param name="sources">The input task-sequence-of-task-sequences.</param>
-    /// <returns>The resulting task sequence.</returns>
+    /// <returns>The resulting, flattened task sequence.</returns>
     /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence of task sequences is null.</exception>
     static member concat: sources: TaskSeq<#TaskSeq<'T>> -> TaskSeq<'T>
 
+    /// <summary>
+    /// Combines the given task sequence of sequences and concatenates them end-to-end, to form a
+    /// new flattened, single task sequence.
```

Comment:
Also these are Combinators, so I guess they do Combine in that technical sense (which we all know is the best sense!)

Everything is always "new" for all TaskSeq operators (hence my campaign for a keyword like yield conveying the fact that stuff comes in, has treatments applied, and flows out, remaining lazy)
flattened is great

I'm not sure 'new' is a useful word; if there is a 'Combines' in the description, then there is something new happening. 
But we are not actually making a thing that is 'new', it's ephemeral

Hence mq quest for a workd like 'renders' or 'yields' to convey that while we are combining things, there is not really anything 'new' except in the sense that any function that does not return its input results in a 'new' value.

Again, just musing...